Tasks/Common/azure-arm-rest/azure-arm-aks-service.ts

```diff
@@ -35,11 +35,14 @@ export class AzureAksService {
         });
     } 
 
-    public getAccessProfile(resourceGroup : string , clusterName : string ): Promise<Model.AKSClusterAccessProfile> {
-        return this.beginRequest(`//subscriptions/{subscriptionId}/resourceGroups/{ResourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{ClusterName}/accessProfiles/clusterUser`,
+    public getAccessProfile(resourceGroup: string , clusterName : string , clusterAdmin : boolean = false): Promise<Model.AKSClusterAccessProfile> {
```

Comment:
NIT: `clusterAdmin` could be an optional parameter. 
Just use `clusterAdmin?: boolean` instead